tests/test_elastic.py

```diff
@@ -425,3 +427,33 @@ def test_search_partial(self):
 
         contribs = search.search_contributor(self.name2.split(' ')[0][:-1])
         assert_equal(len(contribs['users']), 0)
+
+
+class TestSearchExceptions(SearchTestCase):
+    """
+    Verify that the correct exception is thrown when the connection is lost
+    """
+    def setUp(self):
+        if settings.SEARCH_ENGINE == 'elastic':
+            search.search_engine.es = None
+
+    def tearDown(self):
+        if settings.SEARCH_ENGINE == 'elastic':
+            search.search_engine.es = search.search_engine.Elasticsearch(
```

Comment:
Instead of forcing instantiation of `Elasticsearch`, you can reset this to its previous value:

```python
def setUpClass(cls):
    #super....
    cls._es = search.search_engine.es

@classmethod
def tearDownClass(cls):
    # super...
     search.search_engine.es = cls._es
```